src/System.Management.Automation/logging/LogProvider.cs

```diff
@@ -370,6 +377,15 @@ internal override void LogSettingsEvent(LogContext logContext, string variableNa
         {
         }
 
+        /// <summary>
+        /// Provider interface function for logging provider health event.
+        /// </summary>
+        /// <param name="InitFailed"></param>
+        /// <param name="context"></param>
+        internal override void LogAmsiUtilStateEvent(string InitFailed, string context)
```

Comment:
```suggestion
        /// <param name="initFailed"></param>
        /// <param name="context"></param>
        internal override void LogAmsiUtilStateEvent(string initFailed, string context)
```
please also describe the parameters in the comments